shimmingtoolbox/cli/prepare_fieldmap.py

```diff
@@ -14,6 +14,7 @@
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 FILE_OUTPUT_DEFAULT = 'fieldmap.nii.gz'
+MASK_OUTPUT_DEFAULT = 'mask.nii.gz'
```

Comment:
maybe you want to call it mask_fieldmap to distinguish it from other masks in the pipeline (eg: mask_shim)

Reply:
Good point, fixed here: [5c8318e](https://github.com/shimming-toolbox/shimming-toolbox/pull/391/commits/5c8318e6ef77cf7c6167aa48dc94f04c57b68de7)